### ISSUE TITLE
[Oculus] Drakes do not scale off gear

### ISSUE THREAD

**Issue by Dibbarra** [dalaranwow/dalaran-wow#5725]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
All drakes have static damage and health values, regardless of being fresh 80 or ICC25hc geared.
**Expected behaviour**: Tell us what should happen instead
Like the demolishers in Ulduar, drakes should gain HP and deal more damage with abilities as ilvl increases.
**Steps to reproduce the problem**:
Log fresh 80 alt and deal equal damage to your ICC geared main on drakes.
1. 
2. 
3. 

**Include proofs for this behaviour**
 Patch 3.2.0 
(04-Aug-2009): Drakes used by players in The Oculus and The Eye of Eternity will now scale in health and damage according to the rider's average item level.

Patch 3.3.0
Vehicle scaling on the drakes based on the rider's item level has been increased to make them more powerful.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Dibbarra** [comment]
If it wasn't for the fact that the Amber drakes 3rd ability stacks (issue #5429) It would be almost impossible to kill Eregos.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, this issue will be fixed with the next realm maintenance.

**Comment by Lothloryen** [comment]
On live server already.
